topostats/tracing/dnatracing.py

```diff
@@ -121,13 +126,28 @@ def trace_dna(self):
         elif len(self.disordered_trace) >= self.min_skeleton_size:
             self.linear_or_circular(self.disordered_trace)
             self.get_ordered_traces()
+            self.get_trace_heights_and_distances()
             self.linear_or_circular(self.ordered_trace)
             self.get_fitted_traces()
             self.get_splined_traces()
             # self.find_curvature()
             # self.saveCurvature()
             self.measure_contour_length()
             self.measure_end_to_end_distance()
+
+            mask = np.zeros(self.image.shape)
+            for coord in self.ordered_trace:
+                mask[coord[0], coord[1]] = 1
+            Images(
```

Comment:
I realise this is work in progress and suspect this is for development and intended to be removed since plotting of traces is done from `processing.process_scan()`?